example/lib/pages/custom_crs/custom_crs.dart

```diff
@@ -32,12 +32,12 @@ class _CustomCrsPageState extends State<CustomCrsPage> {
     super.initState();
 
     // EPSG:4326 is a predefined projection ships with proj4dart
-    epsg4326 = proj4.Projection('EPSG:4326');
+    epsg4326 = proj4.Projection.get('EPSG:4326');
```

Comment:
This is in the example app. The proj4 API changed from a constructor to a static getter.
I had to change this so the example app could run.